sunbeam-python/sunbeam/provider/local/steps.py

```diff
@@ -132,21 +139,29 @@ def prompt_for_nic(self, console: Console | None = None) -> str | None:
                 agree_nic_up = sunbeam.core.questions.ConfirmQuestion(
                     f"WARNING: Interface {nic} is configured. Any "
                     "configuration will be lost, are you sure you want to "
-                    "continue?"
+                    "continue?",
                 ).ask()
                 if not agree_nic_up:
                     continue
             if nic_state["up"] and not nic_state["connected"]:
                 agree_nic_no_link = sunbeam.core.questions.ConfirmQuestion(
                     f"WARNING: Interface {nic} is not connected. Are "
-                    "you sure you want to continue?"
+                    "you sure you want to continue?",
+                    description=(
+                        "Interface is not detected as connected to any network. This"
+                        " means it will most likely not work as expected."
```

Comment:
Do you think we can inform what does not work as expected? Like external connectivity?

Reply:
Not sure what you intend.

`Interface not connected to a network means there will be no connectivity` (something like that), seems redundant to me? But maybe needs to be clarified to an user